server/secrets.py

```diff
@@ -1,7 +1,7 @@
-MBTA_V3_API_KEY = ''
-POSTGRES_DB = 'newtrains'
-POSTGRES_USER = 'ubuntu'  # No password; unix account auth
-'''
+MBTA_V3_API_KEY = ""
+POSTGRES_DB = "newtrains"
+POSTGRES_USER = "ian"  # No password; unix account auth
```

Comment:
Should this not be ubuntu?

Reply:
Oh, yeah, I thought I configured this to not commit my changes to secrets! (Luckily there are no actual secrets in here)